docs/content/get-started/open-project.md

```diff
@@ -23,9 +27,11 @@ There is a [known issue]({{urlRoot}}/known-issues) in the Unity Editor regarding
 1. Open the **Navigation** window (Unity Editor menu: **Windows** > **AI** > **Navigation**).
 1. Select the **Bake** tab, and then the **Bake** button.
 
-You can verify that the NavMesh has been baked correctly by navigating to **Assets** > **Fps** > **Scenes** > **FPS-SimulatedPlayerCoordinator**, and checking that Unity displays the correct icon.
+You can verify that the NavMesh has been baked correctly by navigating to **Assets** > **Fps** > **Scenes** > **FPS-SimulatedPlayerCoordinator**, and checking that Unity shows a Navmesh object (in most cases the logo will update to the below).
 <img src="{{assetRoot}}assets/navmesh-fixed.png" style="margin: 0 auto; display: block;" />
 
+<%(Callout type="warning" message="
+You may have a number of _warnings_ in the Unity editor console, which you can ignore at this stage. We recommend only showing info and error messages in the Console." )%>
```

Comment:
```suggestion
> You may get a number of warnings displayed in the Unity Editor Console window. You can ignore the warnings at this stage, so you can use the message icons on the right-hand side of the Console window to set it to display only info and error messages.
```
Put as a simple `>`.
Warnings doesn't need to be italics.
Unity Editor Console window (note that's what it is called and all apart from "window" have upper case initials).

Reply:
Note to self - added to new PR